osbs/tekton.py

```diff
@@ -570,10 +570,14 @@ def get_error_message(self):
                                     if message['key'] == 'task_result':
                                         err_message += f"Error in {task_name}: " \
                                                        f"{message['value']};\n"
+                                continue
```

Comment:
What is the purpose of this continue statement? Seems to me like this is the last statement and the execution will continue anway?